pkg/cloud/services/ec2/instances_test.go

```diff
@@ -245,7 +245,8 @@ func TestCreateInstance(t *testing.T) {
 				},
 				Spec: clusterv1.MachineSpec{
 					Bootstrap: clusterv1.Bootstrap{
-						Data: pointer.StringPtr("user-data"),
+						// echo "user-data" | base64
+						Data: pointer.StringPtr("dXNlci1kYXRhCg=="),
```

Comment:
Shouldn't the value in Bootstrap.Data be just a plain string?

Reply:
As in not base64? I dunno if a decision was made at some point to not do that but the bootstrapper's template func encodes it:

https://github.com/kubernetes-sigs/cluster-api-bootstrap-provider-kubeadm/blob/c150edc4f00e11ba1cad905f18d4caff1ad5d1ae/cloudinit/utils.go#L31